rasa/core/training/interactive.py

```diff
@@ -1412,7 +1412,9 @@ async def _plot_trackers(
 
     from networkx.drawing.nx_pydot import write_dot
 
+    output_file = open(output_file, "w", encoding="utf-8")
     write_dot(graph, output_file)
+    output_file.close()
```

Comment:
Doesn't override existing variable, briefer, no closing necessary:
```suggestion
    with open(output_file, "w", encoding="utf-8") as f:
        write_dot(graph, f)
```